### DISCUSSION TITLE
I wrote a series of articles about Storybook in Chinese.

### DISCUSSION THREAD

**Discussion by lala-lee-jobs** [storybookjs/storybook#12796] [Show and tell]
https://ithelp.ithome.com.tw/users/20130417/ironman/3608
It's a series of articles about Storybook.
我在「第 12 屆 iT 邦幫忙鐵人賽」發表完成 玩轉 Storybook 系列文章
主要是參考 Intro to Storybook 及 Design Systems for Developers 這個二個教學指南

**Comment by shilman** [comment]
很棒!
